client/src/components/public/calendar/ConfirmEventType.vue

```diff
@@ -160,13 +166,20 @@ export default {
         ...mapGetters('publicEvent', {
             eventType: getters.GET_EVENT_TYPE,
             publicEvent: getters.GET_PUBLIC_EVENT
-        })
+        }),
+        startDateFormatted() {
+            return moment(this.publicEvent.startDate).format(
+                'dddd, YYYY-MM-DD, HH:mm'
+            );
+        }
     },
     methods: {
         onScheduleEvent() {
             this.$refs.form.validate();
             if (this.formValid) {
-                this.$router.push({ name: 'PublicEventDetails' });
+                this.$router.push({
+                    path: `/${this.eventType.owner.name}/${this.eventType.id}/invitee/details`
```

Comment:
`owner.nickname`, we will with owner nickname
`eventType.slug`
`/eventType.owner.nickname/eventType.slug/invitee/details`